CATALOG.md

```diff
@@ -0,0 +1,42 @@
+# Build Commands Library
+
+Below you'll find a list of build commands contributed from the nativefier community.
+
+If you'd like to add to this catalog, please keep the following in mind:
+
+Library commands should be the strict necessary to make an app work. So, for example,
+
+- Yes to have `--widevine` in Udemy. Yes to `--internal-urls` and `--browserwindow-options` for Outlook Web. Yes to a name and icon for both...
+- ... but let's not have all the other flags that are a matter of personal preference (e.g. `--disable-dev-tools` or `--disk-cache-size`).
+
+* * *
+
+## Outlook Web
+
+### Windows Command
```

Comment:
I wouldn't keep these headings, they add repeated noise, and consume vertical space. Just the example in triple backticks and a list of notes is enough.